### PR TITLE
Move libffi to common installs

### PR DESCRIPTION
**Pull request by benrudolph**
@millerdev this is required now that github3py will be used in our prod environments

### CODE REVIEW

**Comment by benrudolph** [comment]
(has been run on all envs)

File: ansible/roles/common_installs/tasks/main.yml [hunk 1]
```diff
@@ -58,6 +58,7 @@
     - libfreetype6-dev
     - gradle
     - ranger  # cmdline file browser
+    - libffi-dev  # Required for installing github3py
```
Comment (millerdev) [inline ansible/roles/common_installs/tasks/main.yml]: This task is installing lots of stuff that is not needed on many machines. For example, `libffi-dev` (and many others too) is not needed by Riak CS machines. Not a blocker for this PR, but would be nice if we moved a lot of this stuff to the `commcarehq` role.

Comment (benrudolph) [inline ansible/roles/common_installs/tasks/main.yml]: true agreed. this package is pretty small ~300kb so it's not a huge deal, but is nice for managing complexity

Comment (millerdev) [inline ansible/roles/common_installs/tasks/main.yml]: Yeah, size is not remotely the issue. Putting stuff on servers that is not needed is a bad practice for other reasons: it makes maintenance more difficult, can result in implicit/undocumented dependencies over time, and in some cases can introduce security vulnerabilities.

**Pull request merged**